package.json

```diff
@@ -20,12 +22,15 @@
   "homepage": "https://github.com/BelaM80/webpack-demo#readme",
   "devDependencies": {
     "clean-webpack-plugin": "^3.0.0",
+    "concurrently": "^5.3.0",
```

Comment:
`concurrently` is a nice module, but you don't use it here.
Same with `live-server`. I would uninstall bot modules.

Reply:
thanks, I will do that